docs/source/api_reference/forecasting.rst

```diff
@@ -138,6 +138,8 @@ Composition
 
     ColumnEnsembleForecaster
     EnsembleForecaster
+    AutoEnsembleForecaster
```

Comment:
its there :) or what is missing @fkiraly ?

Reply:
ah, it's actually there - forget what I said